docs/_templates/buttons.html

```diff
@@ -0,0 +1,34 @@
+<style>
+    .circle-icon {
+        width: 28px;
+        height: 28px;
+        margin-left: 5px;
+        margin-right: 5px;
+        margin-top: 15px;
+    }
+</style>
+
+<br/>
+<p>
+<a href="https://github.com/radis/radis">
+    <img class="circle-icon"
+         src="/home/sonika/radis/docs/_static/assets/general/button_github.svg"
```

Comment:
It would be better to use relative file paths here, so that the build runs on all machines

Reply:
Also, @anandxkumar the build error in this PR is unrelated to this PR, right ?

Reply:
I have updated and used relative file paths